app/controllers/webhooks/webhooks.controller.js

```diff
@@ -107,6 +118,16 @@ async function toggleActiveWebhook(req, res, next) {
   }
 }
 
+async function resendWebhookMessage(req, res, next) {
```

Comment:
we don't actually have the backend for this yet, but good that we're thinking about the UI for it!

Reply:
yeah we'll need to capture getting this done on the backend! I've removed the button from actually showing up until that's in as for now it would 404 trying to talk to the webhooks service